doc/news/changes.h

```diff
@@ -39,6 +39,12 @@ inconvenience this causes.
 
 <ol>
 
+ <li> changed: The static variable supports_distributed_data has been replaced 
```

Comment:
I think you ought to also say that this variable is part of the vector interface, same for the function name in the next line.